src/Maker/MakeCommand.php

```diff
@@ -58,13 +69,26 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             sprintf('The "%s" command name is not valid because it would be implemented by "%s" class, which is not valid as a PHP class name (it must start with a letter or underscore, followed by any number of letters, numbers, or underscores).', $commandName, Str::asClassName($commandName, 'Command'))
         );
 
+        $this->useStatements = [
+            Command::class,
+            InputArgument::class,
+            InputInterface::class,
+            InputOption::class,
+            OutputInterface::class,
+            SymfonyStyle::class,
+        ];
```

Comment:
Hmm. I don't love this whole "setting on a property" idea. It's very common for us to generate multiple files from a single maker. Would we re-use this property for all the different files we generate, being careful to "clear" it between generating? It's messy :/. 

If we want some nice helper for this, let's create a model class - e.g .`UseStatements`:

```
$useStatements = new UseStatements([...]);
$useStatements->add(...);

echo $useStatements; // returns the final string with line breaks and in the right order.
```